azure/functions/authenticationevents.py

```diff
@@ -119,9 +40,9 @@ def decode(cls,
             try:
```

Comment:
What happens here if the event is not found, maybe an error can be thrown? (This is something we can add to the tests)